python/google/protobuf/json_format.py

```diff
@@ -66,7 +66,7 @@ class ParseError(Exception):
   """Thrown in case of parsing error."""
 
 
-def MessageToJson(message, including_default_value_fields=False):
+def MessageToJson(message, including_default_value_fields=False, indent=None):
```

Comment:
sorry about that, I updated the tests and added a test case to ensure indentation is working when specified.